test/surface/formatter/plugin_test.exs

```diff
@@ -106,4 +110,16 @@ defmodule Surface.Formatter.PluginTest do
       """
     )
   end
+
+  test ":hook directive without any attribute" do
+    assert_formatter_doesnt_change(
+      "sface_files.sface",
+      """
+      <div :hook />
+      """,
+      """
+      <div :hook />
+      """
+    )
```

Comment:
Actually, for `assert_formatter_doesnt_change`, I think the point of it is that you only have to specify the code once. (See `test/surface/formatter_test.exs`)

Can we make that change?